pkg/clusteragent/autoscaling/workload/model/doc.go

```diff
@@ -3,5 +3,4 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-// Package model implements data model structures and helpers for workload autoscaling.
```

Comment:
why does this need to be removed?

Reply:
Artifact of removing and re-adding the file as I changed the const usage